flit_core/bootstrap.py

```diff
@@ -0,0 +1,87 @@
+"""Install flit_core without using any other tools.
+
+Normally, you would install flit_core with pip like any other Python package.
+This script is meant to help with 'bootstrapping' other packaging
+systems, where you may need flit_core to build other packaging tools.
+
+Pass a path to the site-packages directory or equivalent where the package
+should be placed. If omitted, this defaults to the site-packages directory
+of the Python running the script.
+"""
+import argparse
+import sys
+import sysconfig
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from zipfile import ZipFile
+
+from flit_core.wheel import add_wheel_arguments, build_flit_wheel
+
+srcdir = Path(__file__).parent.resolve()
+
+def extract_wheel(whl_path, dest):
+    print("Installing to", dest.resolve())
+    with ZipFile(whl_path) as zf:
+        zf.extractall(dest)
+
+def add_install_arguments(parser):
+    parser.add_argument(
+        '--wheeldir',
+        '-w',
+        type=str,
+        help=f'wheel dist directory (defaults to {srcdir.joinpath("dist").resolve()})',
+    )
+    site_packages = Path(sysconfig.get_path('purelib'))
```

Comment:
nitpick: Please call this `purelib`, site packages is a different concept.

Reply:
changed to `purelib`